model-engine/model_engine_server/infra/gateways/live_cron_job_gateway.py

```diff
@@ -97,14 +99,30 @@ async def list_jobs(
             logger.exception("Got an exception when trying to list the Jobs")
             raise EndpointResourceInfraException from exc
 
+        core_client = get_kubernetes_core_client()
```

Comment:
apparently this fn is used in the `list docker image batch jobs` api call for some reason, feels wrong to me, idk how we didn't catch this before, but oh well

Reply:
Could you like to the code that is the issue? Is the problem that we are breaking abstraction layers?

Reply:
`model_engine_server/domain/use_cases/batch_job_use_cases.py:ListDockerImageBatchJobV1UseCase.execute`

This feels like we're breaking abstraction layers to me at least (e.g. batch jobs and cron jobs should be different), although I guess that broken abstraction gets propagated through the API as well (since the ListBatchJobs has a trigger_id parameter)